packages/astro/src/types/public/config.ts

```diff
@@ -1390,6 +1390,76 @@ export interface AstroUserConfig {
 	 */
 	legacy?: object;
 
+	/**
+	 * @docs
+	 * @name env
+	 * @type {object}
+	 * @default `{}`
+	 * @version 5.0.0
+	 * @description
+	 *
+	 * Holds `astro:env` options.
+	 */
+	env?: {
+		/**
+		 * @docs
+		 * @name env.schema
+		 * @type {EnvSchema}
+		 * @default `{}`
+		 * @version 5.0.0
+		 * @description
+		 *
+		 * An object that uses `envField` to define the data type (`string`, `number`, or `boolean`) and properties of your environment variables: `context` (client or server), `access` (public or secret), a `default` value to use, and whether or not this environment variable is `optional` (defaults to `false`).
+		 * ```js
+		 * // astro.config.mjs
+		 * import { defineConfig, envField } from "astro/config"
+		 *
+		 * export default defineConfig({
+		 *   experimental: {
```

Comment:
Make sure to update the code samples to no longer show this under `experimental`!